docs/execute/force-execute.md

```diff
@@ -1,6 +1,6 @@
 Test items are skipped (or not) based on declarations in tests class, using manifest, or @bug/flaky/missing_feature decorators.
 
-This fact implies an [egg-chicken issue](../edit/egg-chicken-changes.md) if you need to work on a feature. A way to handle it is to use the `-F` option :
+This fact implies an "[egg-chicken issue]"(../edit/egg-chicken-changes.md)[NOTE] if you need to work on a feature. A way to handle it is to use the `-F` option :
```

Comment:
@cbeauchesne calling this out because I deleted the egg-chicken-changes doc. I moved info about how to approach the "chicken or the egg" problem to the [enable-test](https://github.com/DataDog/system-tests/blob/mtoffl01-patch-1/docs/edit/enable-test.md) doc. I could link it here instead, but to be honest I don't know what this force-execute doc is referring to and whether it's relevant? 🤔

Reply:
It's an option that can be used locally, but I discourage using it in CI. You can keep it here, just update the link